spec/factories.rb

```diff
@@ -415,7 +416,7 @@ def r_str
   factory :auth_with_read_and_ppid, class: Api::OpenidConnect::Authorization do
     association :o_auth_application, factory: :o_auth_application_with_ppid
     user
-    scopes %w(openid sub aud profile picture nickname name read)
+    scopes { %w(openid sub aud profile picture nickname name read) }
```

Comment:
Style/PercentLiteralDelimiters: `%w`-literals should be delimited by `[` and `]`.